Cargo.toml

```diff
@@ -55,11 +55,11 @@ hex = "0.4.3"
 hmac = "0.12.1"
 home = "0.5.11"
 http-auth = { version = "0.1.10", default-features = false }
-humantime = "2.1.0"
 ignore = "0.4.23"
 im-rc = "15.1.0"
 indexmap = "2.7.1"
 itertools = "0.14.0"
+jiff = { version = "0.2.3", default-features = false, features = [ "std" ] }
```

Comment:
If we're adopting `jiff`, we may want to consider consolidating on it more generally (ie move off of `time`)

Haven't considered whether that is blocking for this

Reply:
Since `jiff` is already in dependency tree via `gix` this PR shouldn't have big change in build nor binary? Dropping `time` in favor of `jiff` could be done in another PR. Quick grep shows `time` is used in six files and mostly `OffsetDateTime` and RFC3339 time formatting.